pytest_girder/pytest_girder/fixtures.py

```diff
@@ -130,9 +138,40 @@ def server(db, request):
         cherrypy.engine.stop()
         cherrypy.engine.exit()
         cherrypy.tree.apps = {}
+        # This is needed to allow cherrypy to restart on another port
+        cherrypy.server.httpserver = None
         docs.routes.clear()
 
 
+@pytest.fixture
+def server(db, request):
+    """
+    Require a CherryPy embedded server.
+
+    Provides a started CherryPy embedded server with a request method for
+    performing local requests against it. Note: this fixture requires the db
+    fixture.
+    """
+    for server in _yieldServer(request):
+        yield server
+
+
+@pytest.fixture
+def exposedServer(db, request):
```

Comment:
Maybe `boundServer` would be a better name?

Reply:
Renamed.